src/dialog/tooltip/tooltip.directive.spec.ts

```diff
@@ -39,7 +39,7 @@ describe("Tooltip directive", () => {
 	it("should create the tooltip component and tooltip should appear at the top", () => {
 		TestBed.overrideComponent(TooltipTest, {
 			set: {
-				template: "<button ibmTooltip='Hello There' placement='top'>Me</button>"
+				template: "<button ibmTooltip='Hello There' placement='top'>Me</button><ibm-placeholder></ibm-placeholder>"
```

Comment:
Let's make this prettier.
```suggestion
				template: `
					<button ibmTooltip="Hello There" placement="top">Me</button>
					<ibm-placeholder></ibm-placeholder>
				`
```